script/DeployScript.s.sol

```diff
@@ -22,7 +22,7 @@ contract DeployScript is Script {
         vm.startBroadcast(deployer);
 
         // deploy proxy admin contract
-        proxyAdmin = new ProxyAdmin();
+        proxyAdmin = new ProxyAdmin(admin);
```

Comment:
Do we want to keep the old version of the deploy script that uses the old version of OZ?